HelpSource/Classes/String.schelp

```diff
@@ -331,7 +331,7 @@ code::
 subsection:: Searching strings
 
 method::find
-Returns the index of the string in the receiver, or nil if not found. If strong::ignoreCase:: is true, find makes no difference between uppercase and lowercase letters. The strong::offset:: is the point in the string where the search begins. string may be a String or a link::Classes/Char::.
```

Comment:
Some stuff got reverted here by accident. Again, I think you're editing from an older version of SuperCollider.